File: Euler 1/Euler1.java

public class Euler1 {
	private static int max = 100;
	public static void main(String[] args) {
		System.out.println(findSum());
	}
	
	private static int findSum() {
		int total = 0;
		for (int i = 3; i < max; i++) {
			if (i % 3 == 0 || i % 5 == 0) total += i;
		}
		return total;
	}
}

File: Euler 1/Euler1.js

function euler1(){
	var sum = 0;
	for (var x = 0; x<1000; x++){
		console.log(x);
		if (x % 3 == 0 || x % 5 == 0) {
			console.log(x);
			sum += x;
		} 
	}
	return sum
	
}


console.log(euler1());